contrib/wlr-randr/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "wlr-randr"
+pkgver = "0.3.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dwerror=false"]
```

Comment:
this is passed by default via cbuild/util/meson

Reply:
Indeed, I fixed this for wlr-randr and also for the wlroots template which is where I had copied it from.

Reply:
Fixed for the other 2 templates that redundantly pass this as well.